bin/validate_submission.py

```diff
@@ -29,8 +29,18 @@
 
 
 def main():
-    argparse = ArgumentParser()
+    argparse = ArgumentParser(description='Validate an ELOAD by checking the data and metadata format and semantics.')
     argparse.add_argument('--eload', required=True, type=int, help='The ELOAD number for this submission')
+    argparse.add_argument('--validation_tasks', required=False, type=str, nargs='+',
+                          default=EloadValidation.all_validation_tasks, choices=EloadValidation.all_validation_tasks,
+                          help='task or set of tasks to perform during validation')
+    argparse.add_argument('--set_as_valid', action='store_true', default=False,
+                          help='Set the script to consider all validation tasks performed as valid in the final '
+                               'evaluation. This does not affect the actual report but only change if the final '
+                               'evaluation')
```

Comment:
"This does not affect the actual report but only change if the final evaluation.." Is there some more information to be added here?